Cubical/Foundations/CartesianKanOps.agda

```diff
@@ -45,20 +44,30 @@ coei0→0 A a = refl
 coei1→0 : ∀ {ℓ} (A : I → Type ℓ) (a : A i1) → coei→0 A i1 a ≡ coe1→0 A a
 coei1→0 A a = refl
 
+private
+  -- "Interpolation" on the interval. Notice:
+  --     interpolateI i0 i j = i
+  --     interpolateI i1 i j = j
+  --     interpolateI (~ t) i j = interpolateI t j i
+  --     interpolateI t i0 i1 = t
+  --     interpolateI t i i = i
+  -- and more...
+  interpolateI : I → I → I → I
+  interpolateI t i j = (~ t ∧ i) ∨ (t ∧ j) ∨ (i ∧ j)
+
+  -- "Equality" on the interval, chosen for the next definition
+  -- (coei→j), because interpolateI k i j is constant in k on
+  -- eqI i j. Note that eqI i i is not i1 but i ∨ ~ i.
+  equalI : I → I → I
+  equalI i j = (i ∧ j) ∨ (~ i ∧ ~ j)
+
```

Comment:
Why is this marked private? Isn't the point that `interpolateI` is a helpful combinator?

In any case, `interpolateI` is definitely the best name so far.

Reply:
I have no authority here, but... I'd make it public. It seems like a useful one-liner to reuse and there's worse definitions that are public. I'd hate to redefine myself :smiley: